parm/globus/run_doorman.sh.j2

```diff
@@ -68,7 +68,7 @@ fi
 set -e
 
 # Remove the working directory
-#shellcheck ignore=SC2050
+#shellcheck disable=SC2050
```

Comment:
Didn't catch this, but we are templating shell script?

Reply:
Yes, because this shell script will be sent to Niagara to be executed.

Reply:
ok.